### PR TITLE
Adds experiment to know if PATH is common issue.

### PR DESCRIPTION
**Pull request by Lewiscowles1986**
Was the solution to #1840. With thanks to @ItzLevvie

- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Are you working against an Issue? I was working against #1840 

-----

Apologies for my conduct on the ticket. It looks like I'd been working around this PATH entry missing with a few windows installer packages. No idea why it was not added.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-cli/pull/1841)

### CODE REVIEW

File: doc/troubleshooting/README.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ These errors most commonly occur for one of following reasons. Please try out th
     >`Get-AppxPackage microsoft.desktopappinstaller`
 
 2. The App Execution Alias for the Windows Package Manager is disabled. You should enable the App Execution Alias for the Windows Package Manager. Go to `App execution aliases` option in `Apps & features Settings` to enable it.
-3. The App Installer did not automatically add the PATH environment variable. You should add the path environment variable. The value to add is "%userprofile%\AppData\Local\Microsoft\WindowsApps".
+3. The App Installer did not automatically add the PATH environment variable. You should add the path environment variable. The value to add is "%userprofile%\AppData\Local\Microsoft\WindowsApps". You can verify this by running `%LOCALAPPDATA%\Microsoft\WindowsApps\winget` from an administrator powershell. If the command runs then, but not before, then you very likely have a missing PATH environment component.
```
Comment (FixYourCrappyEXPERIMENT) [inline doc/troubleshooting/README.md:98]: > You can verify this by running `%LOCALAPPDATA%\Microsoft\WindowsApps\winget` from an administrator powershell. 

It should be **_For CMD  ,_** not powershell. 
For Powershell  `~\AppData\Local\Microsoft\WindowsApps\winget`

**Review by FixYourCrappyEXPERIMENT — CHANGES_REQUESTED** [review]

File: doc/troubleshooting/README.md (hunk 1 above)
Comment (Lewiscowles1986) [reply doc/troubleshooting/README.md]: My bad. I copied from the wrong window. I've given two examples. One for powershell familiar, one for command-prompt, and labelled as such.

**Comment by FixMSIXLimitations** [comment]
Another Note : ***Admin Shell is unnecessary and requiring such is problematic for MSIX packaged applications until it supports "Machine-wide install" and "Running as another users".***  

What happens when a Standard User opens an Admin Shell via UAC only to find that  winget doesn't run ?! MSIX packaged applications to this date are registered Per-User Only and doesn't support Running as another user i.e  requires User Interactive Login.

This is one of the major limitations of MSIX. 
Winget is a MSIX packaged application. MSI or EXE packaged apps don't have these limitations.

So it's advised to remove Admin Shell requirement.

**Review by denelon — APPROVED** [review]

**Comment by denelon** [comment]
Thanks for the PR! Documentation is often the unsung hero in software.

**Pull request merged**